fmc/src/flow/rt_alias.rs

```diff
@@ -147,7 +156,11 @@ impl RtAliasLayer {
     /// * `hand_off` - HandOff
     pub fn extend_pcrs(env: &mut FmcEnv, hand_off: &HandOff) -> CaliptraResult<()> {
         extend_current_pcr(env, hand_off)?;
-        extend_journey_pcr(env, hand_off)?;
+        match env.soc_ifc.reset_reason() {
```

Comment:
We should clear `RTALIAS_CURRENT_PCR` on update reset before extending into it, yeah?

Reply:
done